engine/src/flutter/impeller/entity/contents/atlas_contents.h

```diff
@@ -39,6 +39,8 @@ class AtlasGeometry {
   virtual BlendMode GetBlendMode() const = 0;
 
   virtual bool ShouldInvertBlendMode() const { return true; }
+
+  virtual std::optional<Rect> GetStrictSrcRect() const { return std::nullopt; }
```

Comment:
Docstring please, just add a "see also" that points to the definition of strict source rects.